model/templates.py

```diff
@@ -84,10 +84,15 @@ def create(self, params):
 
         # not local image: set as remote ISO
         path = path.encode('utf-8')
-        if urlparse.urlparse(path).scheme in ["http", "https", "tftp", "ftp",
-                                              "ftps"]:
+        pathscheme = urlparse.urlparse(path).scheme
+        if pathscheme in ["http", "https", "tftp", "ftp", "ftps"]:
             params["cdrom"] = path
             return self.save_template(params)
+        elif pathscheme == 'file':
+            path = urlparse.urlparse(path).path
+        elif pathscheme == '':
```

Comment:
This elif statement is not needed as it does not do anything.

Also seems this commit is part of the first one. Could you join them together?